vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -473,7 +473,7 @@ void Contribution::OnEntrySaved(
       braveledger_bind_util::FromContributionQueueToString(queue->Clone()));
 
     ledger_->SaveContributionQueue(queue->Clone(), save_callback);
-  } else {
+  } else if (queue->type != ledger::RewardsType::ONE_TIME_TIP) {
```

Comment:
this was added as we don't need to mark queue as one time tip don't have queue. This way errors will not printed in log either